api/main.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
+from flask_restful import Api
+from resources.hems import Hems
+from resources.bems import Bems
+from config import app
+
+api = Api(app)
```

Comment:
Initialization should be written as a function and invoke in `if __name__ == "__main__":`.